rotkehlchen/globaldb/handler.py

```diff
@@ -979,6 +980,70 @@ def add_single_historical_price(entry: HistoricalPrice) -> bool:
 
         return True
 
+    @staticmethod
+    def add_manual_current_price(
+            from_asset: Asset,
+            to_asset: Asset,
+            price: Price,
+    ) -> None:
+        """Adds manual current price and turns previously known manual current price into a
+        historical manual price.
+        May raise:
+        - InputError if some db constraint was hit"""
```

Comment:
Is only possible with the assets right?

Reply:
Also if unique constraint will be hit during turning previous manual current price entry into a historical manual price